canonical-web-and-design/global-nav
# Update blog links to u.c/blog

**Currently blocked until https://github.com/canonical-web-and-design/ubuntu.com/pull/5321 is resolved and released**

# QA

- Check out this feature branch
- Run './run' locally
- Visit [http://127.0.0.1:8300](http://127.0.0.1:8300) in your browser
- Check that the following links take you to the relevant page on www.ubuntu.com/blog (as opposed to blog.ubuntu.com):
  - Resources section:
    - Webinars
    - Video
    - Case studies
    - White papers
    - Blog
  - About section:
    - Press centre
